crates/cli/utils/src/chainspec.rs

```diff
@@ -72,9 +72,15 @@ pub struct GenesisAccount {
 /// to a custom one.
 pub fn chain_spec_value_parser(s: &str) -> Result<ChainSpecification, eyre::Error> {
     Ok(match s {
-        "mainnet" => serde_json::from_str(include_str!("../../res/chainspec/mainnet.json"))?,
-        "goerli" => serde_json::from_str(include_str!("../../res/chainspec/goerli.json"))?,
-        "sepolia" => serde_json::from_str(include_str!("../../res/chainspec/mainnet.json"))?,
+        "mainnet" => {
+            serde_json::from_str(include_str!("../../../../bin/reth/res/chainspec/mainnet.json"))?
```

Comment:
Should we change this to cargo manifest dir? Pretty nested at this point

Reply:
That would definitely make it cleaner.

Reply:
Won't matter after #747